src/ucm/rocm/rocmmem.c

```diff
@@ -176,8 +177,31 @@ static ucs_status_t ucm_rocmmem_install(int events)
     return status;
 }
 
+static int ucm_rocm_scan_regions_cb(void *arg, void *addr, size_t length,
+                                    int prot, const char *path)
+{
+    ucm_event_handler_t *handler = arg;
+    ucm_event_t event;
+
+    ucm_debug("dispatching initial memtype allocation for %p..%p %s", addr,
```

Comment:
there should be some check, e.g https://github.com/openucx/ucx/blob/119a81b0f5569fe41fa6e65a78ae4dca3165ad31/src/ucm/cuda/cudamem.c#L304, to identify potential rocm regions. otherwise all memory is added, including host, which could be inefficient.

Reply:
@yosefe thanks, I added a similar section and it still seems to work